### PR TITLE
Localization in functions.php and more

### PR DESCRIPTION
**Pull request by stefaweb**
This change is [Reviewable](https://reviewable.io/reviews/mailwatch/1.2.0/391)

### CODE REVIEW

File: mailscanner/languages/it.php [hunk 1]
```diff
@@ -381,11 +382,11 @@
...
     'admin12' => 'Amministratore',
     'quarrep12' => 'Report Quarantena:',
-    'senddaily12' => 'Invia Report Giornaliero?',
+    'senddaily12' => 'Invia Report Giornaliero',
     'quarreprec12' => 'Destinaratio Report Quarantena:',
     'overrec12' => 'Sostituire il destinatario del report di quarantena?<BR>(verrà utilizzato il tuo nome utente, se vuoto)',
-    'scanforspam12' => 'Analizzare per Spam?',
-    'scanforspam212' => 'Analizzare le e-mail per Spam?',
+    'scanforspam12' => 'Analizzare per Spam:',
+    'scanforspam212' => 'Analizzare le E-Mail per Spam',
```
Comment (endelwar) [inline mailscanner/languages/it.php]: this shouldn't be capitalized

File: mailscanner/languages/it.php [hunk 2]
```diff
@@ -753,7 +754,7 @@
     'title61' => 'Message Quarantine Report',
     'message61' => 'The variable %s is empty, please set a value in conf.php.',
     'text611' => 'Quarantine Report for %s',
-    'text612' => 'In the last %s day(s) you have received %s e-mails that have been quarantined and are listed below. All messages in the quarantine are automatically deleted %s days after the date that they were received.',
+    'text612' => 'In the last %s day(s) you have received %s E-Mails that have been quarantined and are listed below. All messages in the quarantine are automatically deleted %s days after the date that they were received.',
```
Comment (endelwar) [inline mailscanner/languages/it.php]: this shouldn't be capitalized

File: mailscanner/languages/en.php [hunk 3]
```diff
@@ -381,11 +382,11 @@
     'domainadmin12' => 'Domain Administrator',
     'admin12' => 'Administrator',
     'quarrep12' => 'Quarantine Report:',
-    'senddaily12' => 'Send Daily Report?',
+    'senddaily12' => 'Send Daily Report',
     'quarreprec12' => 'Quarantine Report Recipient:',
     'overrec12' => 'Override quarantine report recipient?<BR>(uses your username if blank)',
     'scanforspam12' => 'Scan for Spam:',
-    'scanforspam212' => 'Scan e-mail for Spam?',
+    'scanforspam212' => 'Scan E-Mail for Spam',
```
Comment (endelwar) [inline mailscanner/languages/en.php]: this shouldn't be capitalized

File: mailscanner/languages/en.php [hunk 4]
```diff
@@ -755,7 +756,7 @@
     'title61' => 'Message Quarantine Report',
     'message61' => 'The variable %s is empty, please set a value in conf.php.',
     'text611' => 'Quarantine Report for %s',
-    'text612' => 'In the last %s day(s) you have received %s e-mails that have been quarantined and are listed below. All messages in the quarantine are automatically deleted %s days after the date that they were received.',
+    'text612' => 'In the last %s day(s) you have received %s E-Mails that have been quarantined and are listed below. All messages in the quarantine are automatically deleted %s days after the date that they were received.',
```
Comment (endelwar) [inline mailscanner/languages/en.php]: this shouldn't be capitalized

**Comment by stefaweb** [comment]
In fact, I checked on Google.

In English, the right "syntax" seems "Email".
For French, it is "E-Mail".
For others languages, don't know.

But I can put what you want. ;)

**Comment by stefaweb** [comment]
Collins dictionary

German : E-Mail
French E-Mail (courriel)
English : E-mail or Email
English US :  E-mail or e-mail
Italian : E-mail or e-mail
Portuguese: E-mail or e-mail
Dutch: E-mail or e-mail

**Comment by stefaweb** [comment]
Done.

**Comment by stefaweb** [comment]
Found an HTML construct error in do_message_ops.php (1159aa7).
Manuel can you check the mod?

File: mailscanner/do_message_ops.php [hunk 5]
```diff
@@ -115,15 +120,15 @@
...
+//echo '</table>' . "\n";
 
-echo '  </td>' . "\n";
-echo ' </tr>' . "\n";
+//echo '  </td>' . "\n";
+//echo ' </tr>' . "\n";
 echo ' </table>' . "\n";
 
-echo '<p><a href="javascript:history.back(1)">Back</a>' . "\n";
+echo '<p><center><a href="javascript:history.back(1)">' . __('back21') . '</a></center><br>' . "\n";
```
Comment (endelwar) [inline mailscanner/do_message_ops.php]: center is a deprecated html5 tag, use `<p style="text-align:center">`

**Comment by endelwar** [comment]
Good catch for open tags. You can remove also commented lines 77-78 (that's where the leftover is coming from)

**Comment by stefaweb** [comment]
Ok for me.

**Comment by stefaweb** [comment]
Should be ok to publish.
A lot of works done.

File: mailscanner/do_message_ops.php [hunk 6]
```diff
@@ -128,7 +128,7 @@
 //echo ' </tr>' . "\n";
 echo ' </table>' . "\n";
 
-echo '<p><center><a href="javascript:history.back(1)">' . __('back21') . '</a></center><br>' . "\n";
+echo '<p style="text-align:center"><a href="javascript:history.back(1)">' . __('back21') . '</a><br>' . "\n";
```
Comment (endelwar) [inline mailscanner/do_message_ops.php]: missing </p>

**Comment by endelwar** [comment]
Reviewed 3 of 14 files at r3, 17 of 24 files at r5, 5 of 6 files at r6, 1 of 1 files at r7, 1 of 1 files at r8.
Review status: all files reviewed at latest revision, 5 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mailwatch/1.2.0/391)*

**Comment by stefaweb** [comment]
Review status: all files reviewed at latest revision, 5 unresolved discussions.

---

*[mailscanner/do_message_ops.php, line 131 at r4](https://reviewable.io:443/reviews/mailwatch/1.2.0/391#-KaXZdLJYKtHOHi07qda-r4-131:-Kabqy_5Q175e5qKgaUC:b-896fix) ([raw file](https://github.com/mailwatch/1.2.0/blob/3926299e30194000b80d3dade6b38b301c796666/mailscanner/do_message_ops.php#L131)):*
<details><summary><i>Previously, endelwar (Manuel Dalla Lana) wrote…</i></summary><blockquote>

center is a deprecated html5 tag, use `<p style="text-align:center">`
</blockquote></details>

Done.

---

*[mailscanner/languages/en.php, line 389 at r1](https://reviewable.io:443/reviews/mailwatch/1.2.0/391#-KaNBmUl54AnCxia4A4k-r1-389:-Kabr-8cv8cE56RrphUq:b-896fix) ([raw file](https://github.com/mailwatch/1.2.0/blob/53af625858a6ac86b7dddb46645bdb54b23ef49b/mailscanner/languages/en.php#L389)):*
<details><summary><i>Previously, endelwar (Manuel Dalla Lana) wrote…</i></summary><blockquote>

this shouldn't be capitalized
</blockquote></details>

Done.

---

*[mailscanner/languages/en.php, line 759 at r1](https://reviewable.io:443/reviews/mailwatch/1.2.0/391#-KaNBmUl54AnCxia4A4k-r1-759:-Kabr-tibga3zLUyauzt:b-896fix) ([raw file](https://github.com/mailwatch/1.2.0/blob/53af625858a6ac86b7dddb46645bdb54b23ef49b/mailscanner/languages/en.php#L759)):*
<details><summary><i>Previously, endelwar (Manuel Dalla Lana) wrote…</i></summary><blockquote>

this shouldn't be capitalized
</blockquote></details>

Done.

---

*[mailscanner/languages/it.php, line 389 at r1](https://reviewable.io:443/reviews/mailwatch/1.2.0/391#-KaNBmUl54AnCxia4A4m-r1-389:-Kabr0mqFqxa9Fgo67sY:b-896fix) ([raw file](https://github.com/mailwatch/1.2.0/blob/53af625858a6ac86b7dddb46645bdb54b23ef49b/mailscanner/languages/it.php#L389)):*
<details><summary><i>Previously, endelwar (Manuel Dalla Lana) wrote…</i></summary><blockquote>

this shouldn't be capitalized
</blockquote></details>

Done.

---

*[mailscanner/languages/it.php, line 757 at r1](https://reviewable.io:443/reviews/mailwatch/1.2.0/391#-KaNBmUl54AnCxia4A4m-r1-757:-Kabr1PhqDSGjFPPofuR:b-896fix) ([raw file](https://github.com/mailwatch/1.2.0/blob/53af625858a6ac86b7dddb46645bdb54b23ef49b/mailscanner/languages/it.php#L757)):*
<details><summary><i>Previously, endelwar (Manuel Dalla Lana) wrote…</i></summary><blockquote>

this shouldn't be capitalized
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/mailwatch/1.2.0/391)*

**Pull request merged**